src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2716,6 +2716,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Public Overrides Function GetRequiredLanguageVersion(diagnostic As Diagnostic) As String
+            ' The driving scenario for this API (upgrading project based on language versions reported in diagnostics)
```

Comment:
@dotnet/roslyn-compiler Can you comment on whether returning `Nothing` is an acceptable behavior for an API that is not yet supported?
Note that throwing did break many IDE tests, so that's why I changed it back to `Nothing`.